app/src/main/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfile.kt

```diff
@@ -28,38 +28,16 @@ import org.jellyfin.apiclient.model.dlna.TranscodingProfile
 class ExoPlayerProfile(
 	context: Context,
 	disableVideoDirectPlay: Boolean = false,
-	isAC3Enabled: Boolean = false,
+	audioDirectPlayCodecs : Array<String>,
+	audioTranscodeTarget : Array<String>,
+	enable4KSupport : Boolean
```

Comment:
This is not correctly formatted.

```suggestion
	audioDirectPlayCodecs: Array<String>,
	audioTranscodeTarget: Array<String>,
	enable4KSupport: Boolean
```